src/tsickle.ts

```diff
@@ -896,20 +984,19 @@ class ExternsWriter extends ClosureRewriter {
         let ctors =
             (<ts.ClassDeclaration>decl).members.filter((m) => m.kind === ts.SyntaxKind.Constructor);
         if (ctors.length) {
+          let firstCtor: ts.ConstructorDeclaration = <ts.ConstructorDeclaration>ctors[0];
           if (ctors.length > 1) {
-            // TODO: Pass to overload rendering parameter builder
-            // For now, we just drop all but the first.
-            // See https://github.com/angular/tsickle/issues/180 .
-            this.debugWarn(ctors[1], 'multiple constructor signatures in declarations');
+            paramNames = this.emitFunctionType(
+                ctors as ts.ConstructorDeclaration[],
+                [{tagName: 'constructor'}, {tagName: 'struct'}]);
+          } else {
+            paramNames = this.emitFunctionType([firstCtor]);
```

Comment:
Does this mean if there is an overloaded constructor, it's not tagged with @constructor?